### PR TITLE
resources: add payment_link

### PR DESCRIPTION
**Pull request by otaviopace**
this closes https://github.com/pagarme/ghostbusters/issues/300

### CODE REVIEW

File: test/pagarme/resources/payment_link.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    end
+
+    should 'be able to cancel a payment_link' do
+      payment_link = PagarMe::PaymentLink.create payment_link_params
+      payment_link.cancel
+      assert_equal payment_link.status, 'canceled'
+    end
+
+    should 'be able to search by short_id' do
+      pl = PagarMe::BankAccount.create payment_link_params
```
Comment (murilohns) [inline test/pagarme/resources/payment_link.rb]: I think it's `PagarMe::PaymentLink` here.

Comment (otaviopace) [reply test/pagarme/resources/payment_link.rb]: OOOpsss

Comment (otaviopace) [reply test/pagarme/resources/payment_link.rb]: Fixed! :slightly_smiling_face:

File: lib/pagarme/resources/payment_link.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module PagarMe
+  class PaymentLink < Model
+    def cancel(params={})
```
Comment (murilohns) [inline lib/pagarme/resources/payment_link.rb]: I think you doesn't need to pass any `params` to cancel a `PaymentLink`, can u remove it?

Comment (otaviopace) [reply lib/pagarme/resources/payment_link.rb]: Yup! I can :)

Comment (otaviopace) [reply lib/pagarme/resources/payment_link.rb]: Fixed!

**Pull request merged**